content/docs/introduction/design-doc.md

```diff
@@ -37,6 +37,7 @@ still relevant.
 | [Prometheus timezones support](https://docs.google.com/document/d/1xfw1Lb1GIRZB_-4iFVGkgwnpwuBemWfxYqFdBm7APsE/edit#) | 2021‑05‑29 | Proposed | |
 | [Moving to goreleaser](https://docs.google.com/document/d/16LOT2wK-jntlU-EFADfaEF3YbKH81U9Zl_PvSu4qVwo/edit#heading=h.bupciudrwmna) | 2021‑06‑05 | Proposed |
 | [Alertmanager Log Receiver](https://docs.google.com/document/d/1Oevu2stHVGAupzmc9C7_wW5nTb_CJ6Ut72viXfve6zI/edit?usp=sharing) | 2021‑06‑10 | Proposed |
+| [Extra HTTP parameters in the blackbox exporter](https://docs.google.com/document/d/1VwqXi2TOb5KXaZY6Iio7411x64pJao3GusX8MqYsJ2g/edit#heading=h.hzsa87ps5uhr) | 2021‑06‑23 | Proposed | |
```

Comment:
```suggestion
| [Extra HTTP parameters in the blackbox exporter](https://docs.google.com/document/d/1VwqXi2TOb5KXaZY6Iio7411x64pJao3GusX8MqYsJ2g/) | 2021‑06‑23 | Proposed | |
```

Reply:
You just to the heading, but I am seeing the others could also use normalization. Can do that after this is merged.